src/app/map/Tile/Item/item.tsx

```diff
@@ -79,10 +83,29 @@ export const DynamicItemTile = ({
     isCenter,
     scale,
   });
+  
+  // Determine if this tile can be dragged
+  const isDraggable = interactive && tileActions?.canDragTile(item.metadata.coordId) === true;
+  const isBeingDragged = tileActions?.isDraggingTile(item.metadata.coordId) === true;
+  const isDropTargetActive = tileActions?.isDropTarget(item.metadata.coordId) === true;
+  
+  // Prepare drag handlers if draggable
+  const dragProps = isDraggable && tileActions ? {
+    draggable: true,
+    onDragStart: (e: React.DragEvent<HTMLDivElement>) => tileActions.dragHandlers.onDragStart(item.metadata.coordId, e),
+    onDragEnd: tileActions.dragHandlers.onDragEnd,
+    style: {
+      opacity: isBeingDragged ? 0.5 : 1,
+      cursor: isBeingDragged ? 'grabbing' : (isDraggable ? 'grab' : 'default'),
+    }
+  } : {};
```

Comment:
same as for the EmptyDynamicTile, make the drag and drop integration as light as possible and just create a helper below.